en_US/durability/durability_introduction.md

```diff
@@ -0,0 +1,88 @@
+# Durable MQTT Sessions and Messages
+
+Starting from release v5.5.x, EMQX contains an embedded durable storage for MQTT sessions and messages. This page introduces the session persistence feature EMQX and how it's capability ensures the resumption of sessions after an EMQX node restart, marking a significant enhancement for IoT applications requiring high reliability.
+
+::: warning Important Notice
+This feature is in the public alpha phase. We advise against its use in production environments containing critical data until further notice.
+
+During the alpha phase, there may be changes to the on-disk representation of MQTT messages and sessions. Consequently, data recorded in one version of EMQX may not be readable after upgrading to a subsequent version. This caution will become obsolete with the final release.
+
+:::
+
+## Understand Client Sessions in EMQX
+
+In EMQX, client sessions facilitate the management of client connections and states within the MQTT broker. Starting from version 5.5, EMQX offers two different implementations for managing the client session: regular client sessions and persistent client sessions.
+
+::: tip
+
+For more information on sessions, see [MQTT Persistent Session and Clean Session Explained](https://www.emqx.com/en/blog/mqtt-session).
+
+:::
+
+### Regular Client Sessions
```

Comment:
wonder if there is a better word than "regular".
maybe "ephemeral" ?

Reply:
Fixed.